src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -218,7 +215,7 @@ protected function getClass(FormBuilderInterface $formBuilder)
     /**
      * @return array
      */
-    protected function getTypes(FormBuilderInterface $formBuilder)
+    private function getTypes(FormBuilderInterface $formBuilder)
```

Comment:
`@return` docblock can be removed, or improved:
```suggestion
    private function getTypes(FormBuilderInterface $formBuilder): array
```

Reply:
docblock will be FormTypeInterface[]

Reply:
Fixed